luatest/assertions.lua

```diff
@@ -535,6 +541,7 @@ function M.assert_error_msg_contains(expected_partial, fn, ...)
     if type(error_msg) ~= "string" then
         error_msg = tostring(error_msg)
     end
+    log.info('Assert error message %s contains %s', error_msg, expected_partial)
```

Comment:
I propose to use the following format:

`` Assert error message `%s` contains `%s` ``

Also, looks like it should be at the very beginning of the function.

Reply:
I've fixed that.